masterlist.yaml

```diff
@@ -710,6 +710,11 @@ plugins:
       - 'NOM3.0.esp'
       - 'NOM3_PT SO.esp'
       - 'NoM_Creatures Loot Standard.esp'
+  - name: 'Vampire Masquerade - A Detailed Overhaul(?: TR)?\.ESP'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/56101' ]
```

Comment:
Please add a trailing slash `/` at the end of the url, so:
`https://www.nexusmods.com/morrowind/mods/56101/`